recipes/oatpp-postgresql/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
   "1.2.0":
     folder: all
+
+
```

Comment:
```suggestion
```

No need to modify this file

Reply:
I modified this line because there was `Unexpected error` and I had to pull force to start checks again.

Reply:
You can just close/reopen the PR in this case.

Reply:
I tried this in other PR and didn't worked. I'll try again when this error will occur